src/components/NotificationModal/index.tsx

```diff
@@ -136,13 +136,13 @@ export const NotificationModal: React.FC<NotificationModalProps> = ({
           onChange={(v: number) => {
             setSelectedTriggerIndex(v);
           }}
-          data-test-id="when-select"
+          data-cy="when-select"
         >
           {triggers.map((t, i) => (
             <Option
-              key={`trigger_${t.payloadResourceIdKey}`}
+              key={`trigger_${t.resourceType}`}
```

Comment:
`resourceType` is also repeated a lot [here](https://github.com/evergreen-ci/spruce/blob/main/src/components/PatchActionButtons/addNotification/PatchNotificationModal.tsx#L95) so I'm still seeing key errors. I think `trigger` would be the appropriate field to use?

Reply:
Thanks! I realized none of these keys individually are unique enough to be used so i created a key that combines a few of them.